specification/devcenter/DevCenter/shared/routes.tsp

```diff
@@ -0,0 +1,20 @@
+import "@typespec/rest";
+import "./models.tsp";
+import "@azure-tools/typespec-azure-core";
+
+using Azure.Core;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace DevCenterService;
+
+interface SharedOperations {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is a custom operation status endpoint."
+  @doc("Get the status of an operation.")
+  @route("/projects/{projectName}/operationstatuses/{operationId}")
+  getProjectOperationStatus is Foundations.GetOperationStatus<{
+    @doc("The DevCenter Project upon which to execute operations.")
+    @path
+    projectName: string;
+  }>;
```

Comment:
Can you make this an alias? I think that will fix the lint diff error

Reply:
didn't fixed, i've reverted to the old way